src/main/java/edu/kit/compiler/JavaEasyCompiler.java

```diff
@@ -102,8 +116,15 @@ public static void main(String[] args) {
             result = Result.Ok;
         } else if (cmd.hasOption("e")) {
             String filePath = cmd.getOptionValue("e");
-
-            result = echo(filePath);
+            try {
+                String content = echo(filePath);
+                System.out.println(content);
```

Comment:
Why do we print the string here and not in `echo`? I would prefer keeping the logic/action in the separate methods and using the main method only for command line parsing and stuff like that.

Reply:
For testing purposes